bin/apisix

```diff
@@ -821,6 +821,11 @@ local function init()
         sys_conf["dns_resolver"] = dns_addrs
     end
 
+    local env_worker_processes = os.getenv("AS_WORKER_PROCESSES")
```

Comment:
how about change `AS_` prefix to `APISIX_`? which is more clear

Reply:
How about `APIX_`?
The prefix generally does not exceed four letters, such as `kubernetes -> kube`, `prometheus -> prom`, `grafana-> gf`